perseo-main/src/main/java/com/telefonica/iot/perseo/LogLevelServlet.java

```diff
@@ -95,11 +103,18 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
             throws ServletException, IOException {
         logger.debug("getting log level");
         synchronized (mutex) {
-            String currentLevel = LogManager.getRootLogger().getLevel().toString();
-            response.setCharacterEncoding("UTF-8");
-            response.setContentType("application/json");
-            response.setStatus(HttpServletResponse.SC_OK);
-            response.getOutputStream().print("{\"level\":\""+ currentLevel +"\"}");
+        	 try {
+	            String currentLevel = LogManager.getRootLogger().getLevel().toString();
+	            response.setCharacterEncoding("UTF-8");
+	            response.setContentType("application/json");
+	            response.setStatus(HttpServletResponse.SC_OK);
+	            response.getOutputStream().print(String.format("{\"level\":\"%s\"}",Encode.forHtmlContent(currentLevel)));
+        	}catch(IOException e) {
+	         	logger.error("IOException in log level");
+	         	response.setStatus(HttpServletResponse.SC_BAD_REQUEST);
+	         	return;
+            }
+
```

Comment:
Indent is not correct.

Intent rule: each bloack add 4 spaces of indent. You can see it all around the existing code. Please, don't use tabs.

Reply:
Fixed in 05982ba4eefed043b201d8b95eb1c9b6190d2673